File: 1-modelagem-codigo-php/src/app/Usuario.php
<?php

namespace App;

class Usuario
{
    private int $id;
    private string $nome;

    /** @var Emprestimo[] */
    private array $emprestimos = [];

    public function __construct(int $id, string $nome)
    {
        $this->id = $id;
        $this->nome = $nome;        
    }

    public function getId(): int
    {
        return $this->id;
    }

    public function getNome(): string
    {
        return $this->nome;
    }

    /** @return Emprestimo[] */
    public function getEmprestimos(): array
    {
        return $this->emprestimos;
    }

    public function addEmprestimo(Emprestimo $emprestimo): void
    {
        $this->emprestimos[] = $emprestimo;
    }
}

File: 1-modelagem-codigo-python/app/usuario.py
from __future__ import annotations
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from emprestimo import Emprestimo


class Usuario:
    def __init__(self, id: int, nome: str):
        self.__id = id
        self.__nome = nome
        self.__emprestimos: list[Emprestimo] = []

    def get_id(self) -> int:
        return self.__id

    def get_nome(self) -> str:
        return self.__nome

    def get_emprestimos(self) -> list[Emprestimo]:
        return self.__emprestimos

    def add_emprestimo(self, emprestimo: Emprestimo) -> None:
        self.__emprestimos.append(emprestimo)